slidedeck/src/slides/03-react-state/05-component-lifecycle.pug

```diff
@@ -13,6 +13,36 @@ section
     li <span class="component red">render()</span>
     li <span class="component red">componentDidMount()</span>
 
+section
+  script(type="text/plain", class="language-jsx").
+    import React from 'react';
+    import axios from 'axios';
+
+    export default class TodoList extends React.Component {
+      state = { 
+        todos: [],
+        loading: false 
+      };
+
+      async componentDidMount() {
+        this.setState({ loading: true });
```

Comment:
Any benefit to this pattern vs initializing state to loading: true?

Reply:
Umm, I thought it was a little more clear, but since you mention it, it might cause a weird initial render. I'll change it up.